app/models/resource_action_workflow.rb

```diff
@@ -55,7 +57,10 @@ def validate_dialog
   end
 
   def create_values
-    create_values_hash.tap { |value| value[:src_id] = @target.id }
+    create_values_hash.tap do |value|
+      value[:src_id] = @target.id
+      value[:request_options] = @request_options if @request_options.any?
```

Comment:
since you already added it as `attr_accessor`, you can reference it (get or set) through `request_options` without explicitly using `@request_options`